### DISCUSSION TITLE
No easy way to export beatmaps?!

### DISCUSSION THREAD

**Discussion by ghost** [ppy/osu#19359] [Ideas] [answered]
I've been trying to figure out a way to export all of my beatmaps from lazer to my computer so that I can back them up, reinstall my OS and import them again, but I haven't found any easier way to do it other than going to edit beatmap screen > file > export. It would obviously be very time consuming to go through every single beatmap considering my ~28 GB collection.

I propose that an _Export/Backup ALL beatmaps_ button should be added into the settings menu here:

![export all button](https://user-images.githubusercontent.com/94078268/180665278-79edc859-75ba-4d3d-b99b-0cbc55bcee3d.png)

Also there could be a button to export a single beatmap set by right clicking it here:

![export button](https://user-images.githubusercontent.com/94078268/180665459-13f53220-9be3-4436-9654-63765d7f94e6.png)

As a side note this made me realize lazer does not show the number of loaded beatmaps anywhere. Personally I'm proud of building my collection and I think many players might feel the same. Although it's a minor detail I think it adds some sense of progress. Besides it makes me smile from time to time. :)

**Comment by frenzibyte** [comment, marked as answer]
Easiest way to backup your files is to copy the osu! folder and then place it back after reinstall.

**Reply by ghost** [reply]
I did think of that, but I was worried it might break something - I once tried that on stable and after pasting it back, one specific map (Illegal paradise) wouldn't work. (I do have to note that there was a warning somewhere saying not to do that.) I suggest that it should be documented, since it's not completely obvious. I think the "today's tips" section when launching the game would be perfect.

today's tip:
You can easily backup your files (such as beatmaps) by copying the entire osu folder!

Thank you for the help. :)
